db/interface.go

```diff
@@ -1,7 +1,5 @@
 package db
 
-import "time"
```

Comment:
nit: is this change needed?

Reply:
Yes, the driver no longer exposes a max time or timeout. To do the same behavior, you need to do a context with a timeout. They document it as it is deprecated and I added some notes here: https://github.com/mongodb/anser/pull/92/files#diff-663485385d1519cc72821f4b6ca656c52b0d1b4bee4d414df910f2a01ddfd9b3R483 and similar functions.

Reply:
I think it's fair to straight up remove it and not keep supporting it similar to the mongodb driver did.